### ISSUE TITLE
Tribal Nations help content not announced

### ISSUE THREAD

**Issue by lfrancis** [usnationalarchives/nara-bia-photos#139]
https://nara-bia-photos.herokuapp.com/tribal-nations

The help button link is accessible but the content is not announced by VoiceOver:

![image](https://user-images.githubusercontent.com/6942589/96000102-8fca7400-0e04-11eb-9115-baf3f0ba3e60.png)
